Sources/Crashes/CrashCollection.swift

```diff
@@ -80,8 +90,10 @@ public final class CrashCollection {
             didFindCrashReports(pixelParameters, processedData) {
                 Task {
                     for payload in processedData {
-                        await self.crashSender.send(payload)
+                        let result =  await self.crashSender.send(payload, crcid: self.crcidManager.crcid)
```

Comment:
`await`ing in a loop can take a long time if we have a lot of `payload` entries, but this code was already behaving this way so I'd recommend we leave it as-is for now.